WordPress/src/test/java/org/wordpress/android/sharedlogin/resolver/SharedLoginResolverTest.kt

```diff
@@ -106,10 +116,23 @@ class SharedLoginResolverTest {
     }
 
     @Test
-    fun `Should dispatch UpdateTokenPayload if access token is NOT empty`() {
+    fun `Should dispatch UpdateTokenPayload if access token is NOT, flags and saved posts are migrated`() {
```

Comment:
I think we might have lost a word after the `"...NOT"`. Should it be, like `"...NOT empty, flags..."` instead?